### PR TITLE
Crud

### PR DESCRIPTION
**Pull request by joshelser**
pydantic is nice!

### CODE REVIEW

File: app/crud/__init__.py [hunk 1]
```diff
@@ -63,7 +94,7 @@ def transaction(session):
         raise
 
 
-def create_entity(session, entity_type, entity):
+def create_entity(session, entity_type, entity, validation_proc):
```
Comment (rymurr) [inline app/crud/__init__.py:97]: how come `validation_proc` came back?

File: app/crud/test_label.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+def test_label():
+    _ = Label(
+        name="label1",
+        condition="user_name = 'josh@sundeck.io'",
+        modified_at=datetime.now(),
+        created_at=datetime.now(),
+        enabled=True,
+        is_dynamic=False,
+    )
```
Comment (rymurr) [inline app/crud/test_label.py:12]: nit: end of line...this will make pre-commit fail

File: bootstrap/009_crud.sql [hunk 3]
```diff
@@ -1,10 +1,10 @@
 
-CREATE OR REPLACE PROCEDURE ADMIN.CREATE_ENTITY(entity object, table_name text, validation_proc text)
+CREATE OR REPLACE PROCEDURE ADMIN.CREATE_ENTITY(table_name text, entity object, validation_proc text)
```
Comment (rymurr) [inline bootstrap/009_crud.sql:2]: i don't think we need table_name anymore. I had planned on `CREATE_LABEL` and `CREATE_PROBE` to call this with the `entity_type` pre-set. May not even need this proc at all.

Comment (joshelser) [reply bootstrap/009_crud.sql:2]: cool cool. Was not an opinionated change. I saw these were misordered after doign a devdeploy and made the sql and py consistent.

**Pull request merged**